docs/src/Atmos/SurfaceFluxes.md

```diff
@@ -0,0 +1,114 @@
+# `SurfaceFluxes`
+
+```@meta
+CurrentModule = CLIMA.SurfaceFluxes
+```
+
+Surface flux functions, e.g. for buoyancy flux, friction velocity, and exchange coefficients.
+
+## `Byun1990`
+
+Compute surface fluxes using the approach in Byun (1990).
+
+### Plots
+
+```@example byun1990
+using CLIMA.SurfaceFluxes.Byun1990
+using Plots, LaTeXStrings
+
+Ri_range = range(-1.2, stop=0.24, length=100)
+scales = [50,200,600,1000,10_000]
+
+z_0 = 1.0
+γ_m, γ_h = 15.0, 9.0
+β_m, β_h = 4.8, 7.8
+Pr_0 = 0.74
```

Comment:
are these all reasonable values?

Reply:
I can confirm that all but `z_0` are reasonable. Looking into `z_0`.

Reply:
To produce Fig 4, I think we can use several values for `z_0`:
```
 70.0
 17.5
  5.83
  3.5
  0.35
```
Need to confirm this (plots are not showing up in the docs for me right now)..

Reply:
that's usually a sign of an error (unfortunately it doesn't give a stacktrace). Easiest option is to just load up the `docs` project in a REPL and copy paste